main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -169,6 +169,7 @@ ReconStandardServicePanel.prototype._populatePanel = function() {
       var radio = $('<input type="radio" name="type-choice">')
       .val(typeID)
       .attr("typeName", typeName)
+      .attr("id","type-choice"+ typeID)
```

Comment:
The `typeID` is provided by the reconciliation and could be any string. For instance, it could contain whitespace, which is forbidden by the specifications (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id). So perhaps it would be better to use the index of the type in the list. You would need to add it as a parameter to the `createTypeChoice` function, but that looks doable.

Reply:
Another approach, which does not require introducing a unique `id`, is to make sure the `<label>` tag wraps both the `input` element and its text.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label.

Reply:
You're right, An easier fix to the already proposed solution could be to add `type.trim()` to this `var typeID = typeof type == "string" ? type : type.id;` for that will fix the white-space issue. I've tested it and it works quite alright

@wetneb what do you think?

Reply:
I don't think trimming will help because the whitespace can be anywhere in the string. Transforming the type ids can also make them non-unique. I'd prefer any of the two solutions above.

Reply:
I just pushed the code for the second approach